project/CassandraSparkBuild.scala

```diff
@@ -125,6 +128,7 @@ object Dependencies {
     val akkaSlf4j           = "com.typesafe.akka"       %% "akka-slf4j"            % Akka           % "provided"  // ApacheV2
     val cassandraThrift     = "org.apache.cassandra"    % "cassandra-thrift"       % Cassandra       guavaExclude // ApacheV2
     val cassandraClient     = "org.apache.cassandra"    % "cassandra-clientutil"   % Cassandra       guavaExclude // ApacheV2
+    val cassandraServer     = "org.apache.cassandra"    % "cassandra-all"          % Cassandra       cassandraServerExclusions // ApacheV2
```

Comment:
I don't think we can agree on this. What do you need from Cassandra server dependency?